packages/cojson-transport-ws/src/index.ts

```diff
@@ -18,6 +18,7 @@ export type CreateWebSocketPeerOpts = {
   role: Peer["role"];
   expectPings?: boolean;
   batchingByDefault?: boolean;
+  retryUnavailableCoValues: boolean;
```

Comment:
Added a flag to handpick the peers to retry.

This is mostly for two reasons:
1. We don't need to retry on storage peers. If a value is unavailable there we can only hope to get it from the server peers
2. I didn't want to introduce the retry on the sync servers.

We could also use a role-based logic (e.g. only "server" peers") but preferred to go for a more explicit route to be sure of where we do retries.